### ISSUE TITLE
SyntaxError: Octal literals are not allowed in strict mode.

### ISSUE THREAD

**Issue by mirague** [SocketCluster/socketcluster#200]
When running Node in strict mode, I get the following error when initialising SocketCluster:

```
/Users/mirague/Projects/my-project/node_modules/socketcluster/index.js:694
    return '\033[' + color + 'm' + message + '\033[0m';
                                               ^^
SyntaxError: Octal literals are not allowed in strict mode.
    at Object.exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:513:28)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/mirague/Projects/my-project/server/websocket-client-listener.js:8:23)
    at Module._compile (module.js:541:32)
```

**Comment by jondubois** [comment]
@mirague What version of Node.js are you using? Do you run node with the `--use_strict` flag?
I can't reproduce it on the latest Node.js v.6.1.0 - That looks like a strange bug since this since `'\033[0m'` is not an octal literal - It is a string. Maybe it's an issue with Node.js?

**Comment by mirague** [comment]
@jondubois I ran this on Node 6.2.2 with `--use-strict` flag on

**Comment by jondubois** [comment]
@mirague I tried on Node.js v6.2.2 on Ubuntu and didn't have any error. What OS are you using?
Are you using coffeescript or typescript or is there anything unusual about your setup?

**Comment by jondubois** [comment]
@mirague Nevermind OK, I can reproduce.

**Comment by jondubois** [comment]
@mirague Ok, this has been fixed in `socketcluster` v4.6.3. Note that you will also need to update the dependency `sc-broker-cluster` to 2.0.3 since there was an issue with the async module in strict mode.
